atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/BackgroundSweeperImpl.java

```diff
@@ -94,13 +95,26 @@ public BackgroundSweeperImpl(LockAwareTransactionManager txManager,
                                  Supplier<Long> sweepPauseMillis,
                                  Supplier<Integer> sweepBatchSize,
                                  SweepTableFactory tableFactory) {
+        this(txManager, kvs, sweepRunner, isSweepEnabled, sweepPauseMillis, sweepBatchSize, tableFactory,
+                Optional.absent());
+    }
+
+    public BackgroundSweeperImpl(LockAwareTransactionManager txManager,
```

Comment:
I would prefer keeping a single constructor and adding the `sweepPerfLogger` parameter to it.